tests/ttnn/sweep_tests/sweeps/sweeps/matmul/short/matmul_user_program_config_mcast_2d.py

```
@@ -18,12 +18,16 @@ class TensorMemoryConfigs(enum.Enum):
     CUSTOM_MEMORY_CONFIG = enum.auto()
 
 
+IN0_INNER_DIM_PER_CORE = 96
```

was 64, it is intentional?

---

yes, 3 tiles test the code better because things can't silently pass if certain CBs are double buffered